### ISSUE TITLE
Varnish is not fully explained - best practices

### ISSUE THREAD

**Issue by sylink** [magento/devdocs#874]
Varnish tends to work out of the box without issue, exactly how it does seems to be a complete mystery where the devdocs are concerned.

Is it possible someone could add some information regarding best practices for theme development/block management with regards to Varnish?

M2 performance tanks quite a bit on sites where this isn't considered, I think due to values like cacheable="false" and perhaps vendors are taking this into consideration when developing modules and so on.

Or if this exists already where is it hiding?

**Comment by xcomSteveJohnson** [comment]
This is a duplicate of issue #339. As we said in [this comment](https://github.com/magento/devdocs/issues/339#issuecomment-170736801), we're waiting on internal resources to improve this topic.

Closing, watch issue 339 for updates.
